assets/resources/badusb/Install_qFlipper_windows.txt

```diff
@@ -0,0 +1,22 @@
+REM made by DEXV#6969
+REM this script auto installs qFlipper
+DELAY 3000
+GUI r
+DELAY 1000
+STRING powershell (new-object System.Net.WebClient).DownloadFile('https://update.flipperzero.one/qFlipper/release/windows-amd64/installer','%TEMP%\qFlipper.exe'); Start-Process "%TEMP%\qFlipper.exe"
+DELAY 500
+ENTER
+DELAY 750
+ALT SPACE
+STRING M
+DOWNARROW
+REPEAT 100
```

Comment:
That looks quite hardcore. We use null-installer it got automated install mode. Can you try to use it instead?

Reply:
@gsurkov may give some hints

Reply:
> That looks quite hardcore. We use null-installer it got automated install mode. Can you try to use it instead?

yea thats for hiding the power shell should i just remove it?

Reply:
yep ill work on it when im home from school @gsurkov i could use your help too

Reply:
yes ill work on it when im home i could use your help @gsurkov

Reply:
> That looks quite hardcore. We use null-installer it got automated install mode. Can you try to use it instead?

I am sorry but i searched a lot for what you asked and i have no idea what you meant by it.

Reply:
@DXVVAY It is possible to launch the installer from the command line.
According to the [NSIS docs](https://nsis.sourceforge.io/Docs/Chapter3.html) (p. 3.2), it's as simple as running 
```
.\qFlipperSetup-64bit-<version>.exe /S
```
for default parameters or 
```
.\qFlipperSetup-64bit-<version>.exe /S /D=C:\Alternative\Install\Path\qFlipper
```
for a customised installation directory. Note that in the second case there should be no quotes in the path even if it contains spaces.